bin/main.rb

```diff
@@ -1,3 +1,65 @@
 #!/usr/bin/env ruby
 
-puts 'Hello World'
\ No newline at end of file
+# rubocop:disable Metrics/BlockNesting
+# rubocop:disable Style/IdenticalConditionalBranches
+
+puts "WELCOME TO JULIAN AND DAVID'S TIC-TAC-TOE"
+puts "Player 1, what's your name?"
+p1 = gets.chomp
+puts "Player 2, what's your name?"
+p2 = gets.chomp
+
+def clear
+  if Gem.win_platform?
+    system 'cls'
+  else
+    system 'clear'
+  end
+end
+
+puts "#{p1} will play Xs. #{p2} will play Os"
+i = 0
+input = 0
+status = false
+while i < 9
+
+  status = false
+
+  board = '+---+---+---+
+| 1 | 2 | 3 |
++---+---+---+
+| 4 | 5 | 6 |
++---+---+---+
+| 7 | 8 | 9 |
++---+---+---+'
+  puts board
+
+  if i.even?
+    puts "#{p1}. Make your move. Select an available spot in the board"
+    while status == false
+      input = gets.chomp
```

Comment:
Your code from 39 to 44 and 47 to 52 is doing the same thing. Please use the DRY principle. 

Instead, you can make one function and call the function every time you need it.